package.json

```diff
@@ -374,7 +374,7 @@
     "cross-spawn": "7.0.3",
     "csstype": "2.6.13",
     "csv-parse": "4.16.0",
-    "cypress": "8.4.1",
+    "cypress": "9.6",
```

Comment:
looks like there is a list of breaking changes when upgrading to 9.x, we might want to review those.

https://docs.cypress.io/guides/references/changelog#9-0-0

Reply:
I don't see anything here that's relevant to us

Reply:
I might be wrong here, but I believe that Cypress is bundled together with NX and in order for us to update out Cypress version we need to update the `@nrwl/cypress` package instead of the one you are updating. In fact, I think the only reason Cypress is a dependency is to extract some Cypress types 🤔 

![Screen Shot 2022-06-02 at 13 51 37](https://user-images.githubusercontent.com/3789875/171644958-4be3f2d1-9fe4-4f30-a903-6da2911915ef.png)

Reply:
There are also some depending on `@nrwl/cypress` for preprocessing, and for running the Cypress tests in nx.

Reply:
Looking at the yarn.lock `nrwl/cypress` is not even dependent on cypress at all. So with this, we are upgrading the cypress version.